src/Components/DeleteAllButton.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import {useDispatch} from 'react-redux'
+import {TodoChanged} from '../features/todo.js';
+import {DoneChanged} from '../features/done.js';
+import {EditableDivChanged} from '../features/editableDiv.js';
+
+const DeleteAllButton = () =>{
+
+    const dispatch = useDispatch();
+
+    const DeleteAllTasks = () =>{
+        dispatch(TodoChanged([]));
```

Comment:
You made a lot of steps, I mean you do it imperatively. So perform it declaratively. It should be done with only one disptach, for example: dispatch(DeleteAllTasks());

And thats it!